src/other/precompile.jl

```diff
@@ -1,3136 +1,1577 @@
 # precompile.jl contains precompilation directives for all methods whose compilation
 # is triggered by running DataFrames tests and takes more than a given threshold
-# For all=true, the threshold is 0.01, for all=false it is 0.2.
+# For all=true, the threshold is 0.01, for all=false it is 0.1.
 # `precompile` calls are generated using SnoopCompile via:
-# using SnoopCompileCore
-# inf_timing = @snoopi tmin=0.01 include("test/runtests.jl")
-# using SnoopCompile
-# pc = SnoopCompile.parcel(inf_timing)
-# SnoopCompile.write("src/other/precompile_tmp.jl", pc[:DataFrames], always=true)
-# sed -i '/categorical/di' src/other/precompile_tmp.jl # Remove CategoricalArrays uses
-# sed -i '/var"/d' src/other/precompile_tmp.jl # var"" is not supported on old Julia versions
-# sed -i '/materialize!/d' src/other/precompile_tmp.jl # Work around an inference bug
-# sed -i '/setindex_widen_up_to/d' src/other/precompile_tmp.jl # Not present on Julia 1.0
-# sed -i '/restart_copyto_nonleaf!/d' src/other/precompile_tmp.jl # Not present on Julia 1.0
```

Comment:
So you didn't use these at all? Any step that can limit the amount of hand editing is good to have.

Reply:
None of signatures that were generated by SnoopCompile.jl matched any of these rules :smile:. Things change fast. (ah - maybe there were some `var"` ones)

But I see that something is failing on 1.0 - I will check what.